<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThemeDialog</class>
 <widget class="QDialog" name="ThemeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>243</width>
    <height>169</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Theme Selection</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QRadioButton" name="theme_christmas_radio_button">
     <property name="text">
      <string>Christmas</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="theme_skull_radio_button">
     <property name="text">
      <string>Skull</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QRadioButton" name="theme_sexy_radio_button">
     <property name="text">
      <string>Sexy</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="theme_ok_button">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="theme_cancel_button">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
